frontend_server/lib/server.dart

```diff
@@ -337,8 +338,13 @@ Future<int> starter(
 
   if (options['train']) {
     final String sdkRoot = options['sdk-root'];
-    options = _argParser.parse(<String>['--incremental', '--sdk-root=$sdkRoot']);
+    final String outputTrainingDill = p.join(
+        Directory.systemTemp.createTempSync('train_frontend_server').path,
```

Comment:
should we attempt to remove temporary directory at the end of training?

Reply:
Yes, of course. Thanks. Done.